src/cli.js

```diff
@@ -17,7 +17,7 @@ var argv = require('yargs').
   describe('seleniumHost','Override the hostname to connect to local webdriver or selenium jar').
   string('seleniumPort').
   describe('seleniumPort','Override the default port used by the local webdriver or selenium jar').
-  boolean('useSeleniumJar').default('useSeleniumJar',undefined).
+  boolean('useSeleniumJar').default('useSeleniumJar',false).
```

Comment:
this will mess the overwriting, chanigng the default in directconnectionprovider is enought